src/app/components/category/category.component.ts

```diff
@@ -14,20 +14,20 @@ export class CategoryComponent implements OnInit {
   constructor(
     private spotifyService: SpotifyApiService,
     private route: ActivatedRoute
-  ) { }
+  ) {}
 
   ngOnInit() {
-    const playlist_id = this.route.snapshot.paramMap.get('id');
-    this.spotifyService.getCategoryPlaylist(playlist_id)
+    const playlistID = this.route.snapshot.paramMap.get('id');
+    this.spotifyService
+      .getCategoryPlaylist(playlistID)
       .subscribe(({ playlists }: any) => {
         const { items } = playlists;
-        this.coverImage = items[0].images[0].url
+        this.coverImage = items[0].images[0].url;
```

Comment:
you will get an error if there will not be any images. Try something like lodash get method, it allows you to safely get some value and if there is no such - returns some default value.
`get(items[0], 'images[0].url', null);`
`items[0]` - target object
`images[0].url` - requred path
`null` - default returned value

Reply:
I'm going to change it to this:
```
getImage(items: any) {
    return items[0] && items[0].images ? items[0].images[0].url : null;
  }
```